tests/kani/AsyncAwait/manual_spawn.rs

```diff
@@ -201,3 +290,22 @@ fn arc_spawn_deterministic_test() {
     );
     assert_eq!(x.load(Ordering::Relaxed), 2);
 }
+
+// #[kani::proof]
+// #[kani::unwind(4)]
+// fn nondeterministic_schedule() {
```

Comment:
Can you uncomment this and select the harness via kani-flags?

Reply:
done